Content.Server/IdentityManagement/IdentitySystem.cs

```diff
@@ -62,11 +62,17 @@ public override void Update(float frameTime)
     // This is where the magic happens
     private void OnMapInit(EntityUid uid, IdentityComponent component, MapInitEvent args)
     {
+        if (component.IdentityEntitySlot is not { } slot)
+        {
+            Log.Error($"Uninitialized IdentityEntitySlot for {ToPrettyString(uid)}.");
```

Comment:
This log _probably_ isn't needed since I think this should be unreachable... but it probably doesn't hurt.